retromock/src/main/java/co/infinum/retromock/CallWrapperFactory.java

```diff
@@ -0,0 +1,38 @@
+package co.infinum.retromock;
+
+import kotlin.coroutines.Continuation;
+import retrofit2.Call;
+import retrofit2.Response;
+
+import java.lang.reflect.Method;
+import java.lang.reflect.ParameterizedType;
+import java.lang.reflect.Type;
+
+class CallWrapperFactory {
```

Comment:
Please make constructor private, as this is util class we don't need to create instances.